db_test.go

```diff
@@ -652,6 +653,29 @@ func TestInCriteria(t *testing.T) {
 	})
 }
 
+func TestContainsCriteria(t *testing.T) {
+	runCloverTest(t, containsPath, func(t *testing.T, db *c.DB) {
+		docs, err := db.Query("contains").Where(c.Field("myField").Contains(4)).FindAll()
+		require.NoError(t, err)
+
+		require.Equal(t, 2, len(docs))
+
+	OuterLoop:
+		for _, doc := range docs {
+			myField := doc.Get("myField").([]interface{})
+			require.NotNil(t, myField)
+
+			for _, elem := range myField {
+				if elem.(float64) == 4 {
+					break OuterLoop
+				}
+			}
+
+			require.Fail(t, "myField does not contain element")
```

Comment:
You could output the element not contained in case of failure, just fore debug

Reply:
Done